electron/src/ipc-communication.ts

```diff
@@ -1,23 +1,56 @@
 import { BrowserWindow, dialog, ipcMain } from 'electron';
 import { IpcRegister } from './ipc-register';
+import imageType from 'image-type';
+import * as fs from 'fs';
+import readChunk from 'read-chunk';
+import * as path from 'path';
 
 class IpcCommunication implements IpcRegister {
     registerIpcHandlers(window: Electron.BrowserWindow): void {
         addSelectDirHandler(window);
+        getImagePathsFromDir(window);
     }
 }
 
 function addSelectDirHandler(window: BrowserWindow | null): void {
```

Comment:
Shouldn't these be class methods?

Reply:
ofc they should and the should be private

Reply:
why they aren't ?

Reply:
not my code m8

Reply:
Because why not

Reply:
fixed in c82b9b8f1203b3c4571b2c61eaeed0dc7a094835